FWCore/Utilities/interface/std_pair_hasher.h

```diff
@@ -0,0 +1,19 @@
+#ifndef FWCore_Utilities_std_pair_hasher_h
+#define FWCore_Utilities_std_pair_hasher_h
+/*
+ tbb::hash was changed to used std::hash which does not have an implementation for std::pair.
+ This hasher is taken from the boost::hash implementation.
+*/
+#include "FWCore/Utilities/interface/zero_allocator.h"
```

Comment:
Shouldn't
```suggestion
#include "FWCore/Utilities/interface/hash_combine.h"
#include <string>
#include <utility>
```
be sufficient?